packages/types/src/METHODS_STORAGE_SUBSTRATE.md

```diff
@@ -0,0 +1,26 @@
+
+---
+
+### substrate
+
+_Well known keys that are not returned by state.getMetadata()_
```

Comment:
Makes sense, but rather don't refer to the call, just say "these are keys that are always available to the runtime implementation". (It is like the RPC calls, i.e. always there, not dynamic)